torch/testing/_internal/common_utils.py

```diff
@@ -1403,12 +1415,7 @@ def __exit__(self, exec_type, exec_value, traceback):
                     self.driver_befores[i],
                     driver_mem_allocated)
 
-                # See #62533
```

Comment:
Please update the comment on line 1406 "failure (except on ROCm, see below)" to remove the ROCm-specific part as we're now aligned with CUDA behaviour

Reply:
Done